.github/actions/AppendToFile/node_modules/.bin/ncc

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
![FailureWithoutNodeModules](https://user-images.githubusercontent.com/85592574/193116873-37470a1b-47f3-44f0-b977-2ea58a08049e.png)

This is what I get if I yank out the whole directory. I then tried keeping the directory, but removing certain folders, but then it would complain about not having one of those folders.

Reply:
@schmittjoseph any resource you could point me to for more info on this? I tried this out on my own test PR and it said that it still couldn't find `@actions/core`, but `npm` did say "added 25 packages, and audited 26 packages in 11s" with no output in stderr. I'm assuming it's probably something simple, I just don't have much context with this.